src/components/Modal/Modal.module.scss

```diff
@@ -1,9 +1,16 @@
+// 1. Modal uses <dialog> element that uses the browser's built-in dialog functionality, so that:
+//    * visibility of the .root element and its backdrop is managed by the browser
+//    * positioning of the .root element and its backdrop is managed by the browser
+//    * z-index of the .root element and its backdrop is not needed as dialog is rendered in browser's Top layer
+// 2. When dialogs are stacked, only the topmost dialog should have a backdrop.
```

Comment:
I don't think so. Another `Dialog` means another layer that prevents interaction with content underneath it. That implies the backdrop should always apply.

Reply:
Let's discuss it on Monday.

Reply:
I reverted the change, so every Modal will open its backdrop. So the background gets darker and darker every time new layer is opened. It is standard behavior.